website/docs/docs/deploy/deploy-environments.md

```diff
@@ -186,6 +190,38 @@ This section allows you to determine the credentials that should be used when co
 </WHCode>
 
 
+## Staging environment <Lifecycle status='beta' />
+
+:::note
+Currently in limited availability beta. Contact support or your account team if you're interested in beta access.
+:::
+
+Staging environments are useful ways to grant developers access to deployment workflows and tools while controlling access to production data. They are configured with their own long-living branch (for example, `staging`) that may be very similar to `main` in many ways while potentially limiting the data the developers can access. 
+
+Ideally, the workflows would move upstream from the Development environment -> Staging environment -> Production environment with developer branches feeding into the staging branch, then ultimately `main`. In many cases, the `main` and `staging` branches will be identical after a merge and remain until the next batch of changes from the `development` branches are ready to be elevated. We recommend setting branch protection rules on `staging` similar to `main`.
+
+### Create a staging environment 
+
+In the dbt Cloud, navigate to **Deploy** -> **Environments** and then click **Create Environment**. Select **Deployment** as the environment type. The option will be greyed out if you already have a development environment.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/create-staging-environment.png" width="85%" title="Create a staging environment" />
+
+
+Follow the steps outlined in [deployment credentials](#deployment-connection) to complete the remainder of the environment setup.
+
+We recommend that the data warehouse credentials be for a dedicated user or service principal.
+
+### Why use a staging environment
+
+Staging environments provide a layer of separation between you development and production deployment environment. It isolates the prod and pre-prod data, which aids in protecting sensitive information and limiting access to large data sets. Developers can edit job definitions in the staging environment while production is limited to only those who need access. It also helps to have additional verification steps between development and deployment. 
+
+Let's say you have `Project A` and `Project B` and that `Project B` is downstream of `Project A` with cross-project refs configured in the models. With a staging environment configured in `Project A`, when developers are working in the IDE for `Project B`, those refs will resolve to the staging environment of `Project A`, rather than production. You'll get the same results with those refs when jobs are run in the staging environment (or other non-production deployments). Only the production environment will reference the production data, isolating the environment, reducing need to duplicate projects. 
```

Comment:
```suggestion
Let's say you have `Project A` and `Project B` and that `Project B` is downstream of `Project A` with cross-project refs configured in the models. When developers are working in the IDE for `Project B`, those refs will resolve to the staging environment of `Project A`, rather than production. You'll get the same results with those refs when jobs are run in the staging environment (or other non-production deployments). Only the production environment will reference the production data, isolating the environment and reducing the need to duplicate projects. 
```